spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -1236,6 +1236,45 @@ def function(arg1, arg2, arg3, arg4, arg5, arg6, arg7)
       expect(File.readlines('.rubocop_todo.yml')[2]).to match(/# using RuboCop version .*/)
     end
 
+    describe 'when --no-exclude-limit is given' do
+      before do
+        Array.new(16) do |i|
```

Comment:
`Array.new(16)` creates an array with 16 `nil` elements. So the block value of `i` is always `nil`. Is this the expected behavior of `16.times`?

```suggestion
        16.times do |i|
```

Reply:
And, does `RuboCop::Options::DEFAULT_MAXIMUM_EXCLUSION_ITEMS + 1` resolve magic number `16`?

Reply:
Indeed. At first I used `Array.new` based on `Performance/TimesMap` because I was using the return value, but now I don't use it, so `Integer#times` is more appropriate 👌

Reply:
> And, does `RuboCop::Options::DEFAULT_MAXIMUM_EXCLUSION_ITEMS + 1` resolve the magic number `16`?

That's the hard part.
If it was the only place we use this value, it would be fine, but we need to check that `exampleN.rb` appears 16 times in .rubocop_todo.yml after this, but I think it would be pretty complicated code to try to reference that value there. The current simple implementation seems to have its advantages in terms of clarity.
What do you think of this?

Reply:
It would be better whether `16` could be clarified. Please see https://github.com/rubocop/rubocop/pull/11036/files#r985230949.